src/wizard/index.tsx

```diff
@@ -54,7 +55,7 @@ export default function Wizard({
   const isLastStep = actualActiveStepIndex >= steps.length - 1;
 
   const navigationEvent = (requestedStepIndex: number, reason: WizardProps.NavigationReason) => {
-    trackNavigate(actualActiveStepIndex, requestedStepIndex, reason);
+    trackNavigate(actualActiveStepIndex, requestedStepIndex, reason, funnelId);
```

Comment:
Not blocking but I would consider using named parameters vs positional. Makes it easier to read functions calls IMO.

Reply:
This is likely to change in the future anyway so I can re evaluate it again then :)